gems/aws-sdk-core/sig/seahorse/client/response.rbs

```diff
@@ -0,0 +1,26 @@
+module Seahorse
+  module Client
+    # <!--
+    # RBS does not support Delegator.
+    # the behavior is mimicked `Seahorse::Client::Response` as much as possible.
+    # -->
+    module DelegatorMethodsForResponse[DATA]
```

Comment:
This module does not actually exist.

In RBS, it's not possible to represent the behavior of [Delegator](https://docs.ruby-lang.org/en/3.2/Delegator.html), so we can't make the return value of the operations method group `Seahorse::Client::Response`.
Instead, we define a non-existent module called `DelegatorMethodsForResponse` to represent the methods defined in `Seahorse::Client::Response`.
By including `DelegatorMethodsForResponse` in the data class, we can represent the actual behavior.

example:
```rbs
module Aws
  module Lambda
    module Types
      class DestinationConfig
        include Seahorse::Client::DelegatorMethodsForResponse[DestinationConfig]

        attr_accessor on_success: OnSuccess

        attr_accessor on_failure: OnFailure
      end
    end
  end
end
```

Moreover, `DelegatorMethodsForResponse` cannot be an interface. An interface does not allow for duplicate methods. The existence of `Aws::Lambda::DestinationConfig#on_success` would result in a method duplication, which is why it cannot be handled as an interface.